circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  timezone:
+    America/Los_Angeles
+
+test:
+  override:
+    - echo "No tests are needed"
```

Comment:
Definitely something I want to add soon! I don't think I have too much time to concentrate on tests yet since I would ideally like to release the typeahead feature as soon as possible, and I'll be changing a couple things for then. I just wanted to begin this so @ycp217 and whoever else can start improving this structure.